draft-ietf-moq-transport.md

```diff
@@ -442,6 +442,29 @@ constrain the information in these fields, for example by restricting them to
 UTF-8. Any specification that does needs to specify the canonicalization into
 the bytes in the Track Namespace or Track Name such that exact comparison works.
 
+### Subscription Identifiers
+
+To optimize wire efficiency, some MoQT messages refer to a track by a numeric
+identifier, rather than the Full Track Name. When a subscriber makes a SUBSCRIBE
+or FETCH request for a track, it associates a Subscriber-Chosen Subscription ID
+(SSID) with the request. SSIDs are scoped to a single MoQT session.  They are
+even; they start at 0 and increase by 2 and have a maxmium value of 2^62-2.  The
+publisher limits the number of requests a subscriber can make using the
+MAX_SUBSCRIPTION_ID SETUP parameter or MAX_SUBSCRIPTION_ID control message.
+
+MoQT also allows the publisher to choose an identifier for each track, called
+the Publisher-Chosen Subscription ID (PSID). PSIDs are odd and have a maximum
+value of 2^61-1. The same identifier MUST NOT be assigned multiple tracks in the
+same session.  PSIDs are required when the publisher initiates the subscription
+(TODO: delete this; PUBLISH message is forthcoming) and are optional when the
```

Comment:
Making it optional is ok, but it seems cleaner if every subscription has both, regardless of who initiated it.

Reply:
I'm thinking the optional nature makes it easier for publishers who don't care about this.

Reply:
I am inclined of keeping it as non optional as well.

Reply:
There is a contingent of folks in the working group who do not want this feature as it is not necessary in their application.  Only publishers that have such an ID should be need to set it.  We can mark this as an open question in the PR.

Reply:
I rewrote this bit to say that PSID is explicitly optional.  Will add to a list of items to discuss further.